app/alert/alert_service.ts

```diff
@@ -0,0 +1,18 @@
+import { Subject } from "rxjs";
+
+export type Alert = {
+  message: string;
+  type: AlertType;
+};
+
+export type AlertType = "error" | "success";
+
+export class AlertService {
+  alerts = new Subject<Alert>();
+
+  show(alert: Alert) {
```

Comment:
nit: it might be nice to have "error" and "success" methods here which just take strings to make this easier to use all over

Reply:
Done!